docs/release-notes/2.38.0.md

```diff
@@ -0,0 +1,209 @@
+# Percona Monitoring and Management 2.38.0
+
+| **Release date:** | Jun 02, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+ --->
+
+
+## Release Highlights
+
+### Important security updates
+
+**RHEL 9**
+
+As security is paramount to us, starting with PMM 2.38.0, we have laid a solid and modernized enterprise-grade foundation for PMM. We have migrated our base operating system on which we build our base image from Enterprise Linux 7 ( CentOS 7) to Enterprise Linux 9 (EL9). Thus starting with PMM 2.38.0, we have made PMM publicly available on EL9 (Oracle Linux 9) as an early preview.
```

Comment:
@EvgeniyPatlan, @talhabinrizwan  can you recheck here + confirm the proper names of the used OS? it looks like we mixing different names. (EL,  Oracle Linux ... )

Reply:
Enterprise Linux - we used Oracle Linux - the free version and not Enterprise

Reply:
Please replace **RHEL 9** and **Enterprise Linux 9** with **OracleLinux 9**. 
And IMHO skip **EL9**, instead use **OracleLinux 9**

Reply:
Done.